packages/core/package.json

```diff
@@ -121,6 +121,7 @@
   "dependencies": {
     "@commerce-apps/raml-toolkit": "^0.4.2",
     "@keyv/redis": "^2.1.1",
+    "quick-lru" : "^5.1.1",
```

Comment:
3PP for quick-lru https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qB0000000hBDRIA2/view